src/plan_partialize.c

```diff
@@ -167,19 +167,20 @@ partialize_agg_paths(RelOptInfo *rel)
  * simple filter (e.g., HAVING device > 3). In such cases, the HAVING clause is
  * removed and replaced by a filter on the input.
  */
-void
+bool
 ts_plan_process_partialize_agg(PlannerInfo *root, RelOptInfo *input_rel, RelOptInfo *output_rel)
 {
 	Query *parse = root->parse;
-
+	bool partials_found = false;
 #if PG10_GE
 	Assert(IS_UPPER_REL(output_rel));
 #endif
 
 	if (CMD_SELECT != parse->commandType || !parse->hasAggs)
-		return;
+		return partials_found;
```

Comment:
```suggestion
		return false;
```

Suggest explicit return value to avoid having to check what the initialized value of `partials_found` is and to avoid accidental bugs if the default value changes.

Reply:
Btw, is it really correct to check `parse->commandType` above, or even necessary given the `hasAggs` check?

Reply:
partial aggs are supported only with SELECT. commandType could be inserts/updates etc.